### DISCUSSION TITLE
recommended version of python?

### DISCUSSION THREAD

**Discussion by sain0722** [idom-team/idom#1097] [Question] [answered]
I'm just about to get started. However, I couldn't find any discussions about Python versions anywhere, so I'm curious if I can use just any version. I'm planning to use a version above 3.7, but is there anything like not being able to use a version above 3.10?

**Comment by sain0722** [comment]
I found some info about it on "[contributor guide](https://reactpy.dev/docs/about/contributor-guide.html#contributor-guide)" 
Python >= 3.9, Is it required?

**Reply by Archmonger** [reply, marked as answer]
That information is available [on PyPI](https://pypi.org/project/reactpy/).

We currently support 3.9+.

**Comment by saipriyaan** [comment]
Hello @sain0722,
Python version depends on the task you are planning to do with it, like Flask supports >=3.7 and reactpy >=3.9. Be sure to check pypi website for checking the versions. It is recommended to use the latest version to start.
Thanks/
